### ISSUE TITLE
Banned for not valid reasons  User ~ "BoyBanana" 06/07/2019

### ISSUE THREAD

**Issue by huynhjordan** [CreeperHost/CreeperHostGui#81]
I am very disappointed about the staffing of this chat. Simply put I was discussing with other players on debate questions, trying to gain insight on others perspectives and inputs about certain topics. I have to strongly emphasise that NO ONE, as in NO ONE in any way, shape or form was offended at the slightest, needless to say this judgement of 'exilement' from the 'global chat' was somewhat unfair and oppressive. The staff of this chat acted like they were above everyone else, deeming what was offensive and within the parameters of the chat; EVERYONE was engaging in conversation, heck I even aided others that needed help with the mod (answering queries they had). It is defiantly unfair for the staff to deem my actions as a misdemeanour, with their one sided view. I cannot stress how immensely oppressive and closed minded the staff of the chat are, as shown by their unfair actions i.e. 'Banning me because they "thought that my discussions triggered people"'; even through there is a lack of evidence to support their argument. For the record if my discussion was so called "triggering" people I would have stopped and changed the topic, It is not my intent to "anger individuals" rather to merely attain their perspectives and opinions. I demand my banning ("BoyBanna") from the global chat to be rectified, and thus reinstating me back into the chat. 

Cordially, BoyBanna ~ 07/07/2019

**Comment by huynhjordan** [comment]
@Draylynn

**Comment by CH-Paul-T** [comment]
Good morning,

I kindly request you refrain from demanding any action taken by third parties providing free services.

As I, myself, personally, are the only person to have done any manual bans today, I can confirm with 100% accuracy any bans undertaken today where 100% valid and will not be overturned.

Kindest regards,
Paul

**Comment by Draylynn** [comment]
This was a ban from yesterday, I remember it clearly.

**Chat moderators are there to enforce the rules and keep the chat on topic.**

Several times the discussion was asked to stop and switch back to the Minecraft topic, as the discussed topic(s) often have a habit of getting out of hand very fast (as I called them "trigger topics"). The adult-orientated and potentially sensitive topics were also not suitable for discussion in a Minecraft chatroom (PG7+), with over 1000 users in the chat at the time, guaranteed to have some young eyes watching (sometimes younger than 7 years old).
You were on the fence the moment I warned, you was not reading the chat properly, and apparently, still don't understand fully what happened: For the record, no one was triggered, I never said anyone was... I even pointed this out in chat and explained why.

- The user was discussing an adult-orientated topic.
- The user was repeatedly, and politely, asked to return to the Minecraft topics: With a full explanation as to why.
- The user proceeded to argue with the chat moderator(s).
- The user was banned.

Under the circumstances, I recommend taking some history lessons, having a gander around Google and other search engines, to further understand the meaning of "oppressed", "communist" and any other terms you wrongly threw at our mods. Additionally, a lesson in netiquette wouldn't go amiss, as it appears to be severely lacking.
